nengo/spa/compare.py

```diff
@@ -18,14 +18,12 @@ class Compare(Module):
         Number of neurons to use in each product computation
     output_scaling : float
         Multiplier on the dot product result
-    radius : float
-        Effective radius for the multiplication.  The actual radius will
+    input_magnitude : float
+        Effective input_magnitude for the multiplication.  The actual input_magnitude will
         be this value times sqrt(2)
-    direct : boolean
-        Whether or not to use direct mode for the neurons
     """
     def __init__(self, dimensions, vocab=None, neurons_per_multiply=200,
-                 output_scaling=1.0, radius=1.0, direct=False,
+                 output_scaling=1.0, input_magnitude=1.0,
```

Comment:
Is the output_scaling factor necessary? @tcstewar thoughts? (I removed it in my version of the compare network).